packages/dds/merge-tree/api-report/merge-tree.legacy.alpha.api.md

```diff
@@ -77,7 +77,7 @@ export abstract class BaseSegment implements ISegment {
     // (undocumented)
     splitAt(pos: number): ISegment | undefined;
     // (undocumented)
-    abstract toJSONObject(): any;
+    abstract toJSONObject(): unknown;
```

Comment:
Not sure if this is better than suppressing the errors for any json stuff...

Reply:
If it affects the API surface, we should suppress the error to prevent breaking consumers. We should fix other violations where they can be fixed without too much effort, but the primary goal is to prevent new violations of our rules. So feel free to disable rules anywhere it would break the API, or if it would require substantial refactoring to accommodate the changes. We should leave TODO comments in any places where we disable though.

Reply:
+1 to commenting wherever we disable so we don't lose the context and don't normalize suppression.